### PR TITLE
delete old Admin and Hosts

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
cleaning up old code that isnt used.  also transferred google maps components from legacy to shared so both folders are the same
## How to Test
- [ ] Visit site/admin/host sign up
- [ ] Verify site/admin/host sign up works

Which devices did you test on?
- [ ] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/legacy/shared/GoogleMaps/GoogleMaps.tsx [hunk 1]
```diff
@@ -8,8 +8,6 @@ const { GOOGLE_MAPS_KEY } = SETTINGS;
 import GoogleMapsContainer from './GoogleMaps.container';
 import AudioLoading from 'legacy/shared/loading/AudioLoading';
 
-import { useDebounce } from 'utils/hooks';
```
Comment (woeltjen) [inline src/legacy/shared/GoogleMaps/GoogleMaps.tsx]: This looks like it unintentionally reverts a change from #292

Comment (tc) [reply src/legacy/shared/GoogleMaps/GoogleMaps.tsx]: @kcvan bump

Comment (kcvan) [reply src/legacy/shared/GoogleMaps/GoogleMaps.tsx]: ah good catch, let me actually close this PR and make a smaller one for just deleting admin and host
